libbpf-rs/src/program.rs

```diff
@@ -570,6 +584,21 @@ impl From<u32> for ProgramAttachType {
             x if x == SkReuseportSelect as u32 => SkReuseportSelect,
             x if x == SkReuseportSelectOrMigrate as u32 => SkReuseportSelectOrMigrate,
             x if x == PerfEvent as u32 => PerfEvent,
+            x if x == KprobeMulti as u32 => KprobeMulti,
```

Comment:
Could we use num-derive for this kind of type casting ?

Reply:
I believe we arrived here because we wanted to trim down non-essential dependencies, see https://github.com/libbpf/libbpf-rs/pull/786